### ISSUE TITLE
ScreenReaderContent and React DND

### ISSUE THREAD

**Issue by buckett** [instructure/instructure-ui#965]
This is a bug in the popular React DND library https://github.com/react-dnd/react-dnd/issues/2068 that means that if the item being dragged contains an absolute positioned element then the preview may be wrong when dragged. The Instructure UI `<ScreenReaderContent>` uses absolute positioning to hide the content from browsers but make it read by screen readers.

I'm not suggesting the `<ScreenReaderContent>` should be changed, but just reporting this incase anyone else runs into the problem.

**Comment by matyasf** [comment]
Thanks for the report, we will likely not fix this but its nice that if someone searches for this sees this ticket.
